### PR TITLE
[4.1] table id

### PR DESCRIPTION
**Pull request by brianteeman**
This pr fixes various copy paste errors with the id of the table and removes some ambiguity 

code review

### CODE REVIEW

File: administrator/components/com_banners/tmpl/banners/default.php [hunk 1]
```diff
@@ -47,7 +47,7 @@
 						<?php echo Text::_('JGLOBAL_NO_MATCHING_RESULTS'); ?>
 					</div>
 				<?php else : ?>
-					<table class="table" id="articleList">
+					<table class="table" id="bannersList">
```
Comment (Quy) [inline administrator/components/com_banners/tmpl/banners/default.php]:
```suggestion
					<table class="table" id="bannerList">
```

Comment (brianteeman) [reply administrator/components/com_banners/tmpl/banners/default.php]: they're all a mix of singular and plural - I dont really care as long as the noun is correct

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on e1ef9b06c81553cc321ea8d5f18a2ea9085bcd95<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36590">issues.joomla.org/tracker/joomla-cms/36590</a>.</sub>

**Comment by PhilETaylor** [comment]
I have tested this item :white_check_mark: successfully on e1ef9b06c81553cc321ea8d5f18a2ea9085bcd95<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36590">issues.joomla.org/tracker/joomla-cms/36590</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36590">issues.joomla.org/tracker/joomla-cms/36590</a>.</sub>

**Comment by brianteeman** [comment]
rebased to 4.1

**Comment by bembelimen** [comment]
Thx

**Comment by brianteeman** [comment]
Thanks for merging

**Pull request merged**